src/python/pants/backend/shell/shunit2_test_runner.py

```diff
@@ -0,0 +1,244 @@
+# Copyright 2021 Pants project contributors.
```

Comment:
This required some non-trivial changes from example-plugin repo. Will call them out.